.testcaferc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "browsers": ["chrome", "firefox"],
+    "skipJsErrors": true
```

Comment:
For anyone's interest:

https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-e---skip-js-errors

@ssbelle Is there a specific reason why you feel we need to do this? We can always remove it down the road, but I'm curious about your intentions here.

Reply:
My memory was that it made safari work without errors, but I've since  upgraded the package version and I don't see that error now. Removed